docs/en-US/component/select.md

```diff
@@ -117,45 +117,47 @@ If the binding value of Select is an object, make sure to assign `value-key` as
+| placeholder           | placeholder                                                                                                      | string                                     | —                   | Select           |
+| filterable            | whether Select is filterable                                                                                     | boolean                                    | true / false        | false            |
+| allow-create          | whether creating new items is allowed. To use this, `filterable` must be true                                    | boolean                                    | true / false        | false            |
+| filter-method         | custom filter method                                                                                             | function                                   | —                   | —                |
+| remote                | whether options are loaded from server                                                                           | boolean                                    | true / false        | false            |
+| remote-method         | custom remote search method                                                                                      | function                                   | —                   | —                |
+| remote-show-suffix    | in remote search method show suffix icon                                                                         | boolean                                    | true / false        | false            |
+| loading               | whether Select is loading data from server                                                                       | boolean                                    | true / false        | false            |
+| loading-text          | displayed text while loading data from server                                                                    | string                                     | —                   | Loading          |
+| no-match-text         | displayed text when no data matches the filtering query, you can also use slot `empty`                           | string                                     | —                   | No matching data |
+| no-data-text          | displayed text when there is no options, you can also use slot `empty`                                           | string                                     | —                   | No data          |
+| popper-class          | custom class name for Select's dropdown                                                                          | string                                     | —                   | —                |
+| popper-options        | Customized popper option see more at [popper.js](https://popper.js.org/docs/v2/)                                 | object                                     | —                   | —                |
```

Comment:
## Self Review

add `popper-options` props description

Reply:
Try to execute a format to see if the indentation will be the same as the original

Reply:
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/27342882/221385684-e8100d33-f7db-4f74-9cbf-5ebfbcaa3cc6.png">

I ran `npm run format` . There are no existing changes.

Reply:
ok.